src/library/scala/math/BigInt.scala

```diff
@@ -382,7 +382,7 @@ final class BigInt(val bigInteger: BigInteger)
    *  @param step   the distance between elements (defaults to 1)
    *  @return       the range
    */
-  def until(end: BigInt, step: BigInt = BigInt(1)) = Range.BigInt(this, end, step)
+  def until(end: BigInt, step: BigInt = BigInt(1)): Seq[BigInt] = Range.BigInt(this, end, step)
```

Comment:
```
[error]  * method until(scala.math.BigInt,scala.math.BigInt)scala.collection.immutable.NumericRange#Exclusive in class scala.math.BigInt has a different result type in current version, where it is scala.collection.immutable.Seq rather than scala.collection.immutable.NumericRange#Exclusive
[error]    filter with: ProblemFilters.exclude[IncompatibleResultTypeProblem]("scala.math.BigInt.until")
```

This is the wrong result type; it should be `NumericRange.Exclusive[BigInt]` apparently.

Reply:
I see. I'll fix it.

Reply:
I've fixed this mistake in https://github.com/scala/scala/commit/72a1925179aa1d91b6eb7af4c907e15455e57bc0#diff-90f5c973f0df34163bc300526201206dR385

Reply:
@hrhino Since I've fixed the problem mentioned, could you review this PR again?

Reply:
`All checks have passed` is shown now.  @hrhino If there are no other problems, could you merge it ?